### PR TITLE
save logs in storage account

### PR DESCRIPTION
**Pull request by jimmy-cyr**
save log in storage account

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -175,13 +175,22 @@ jobs:
             terraform apply -auto-approve -var-file="$TERRAFORM_ENV_CONFIG_FILE"
 
 workflows:
+  pull-request:
```
Comment (arahimee) [inline .circleci/config.yml:178]: just a food for thought: we could also use the `filter` on each job to include or exclude certain branches. That way you would have only one workflow. Ex: https://support.circleci.com/hc/en-us/articles/115015953868-Filter-workflows-by-branch-

Comment (arahimee) [reply .circleci/config.yml:178]: oh I just saw that it was using filters and you removed it :D

File: FSharpTemplate/Terraform/na-ci.terraform.tfvars [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
 
-environment             = "Ayuda Dev"
-environment_suffix      = "-NA-CI"
-resource_group_name     = "ayudalabs-na-01"
-resource_group_location = "North Central US"
-service_plan_name       = "ayudalabs-na-ci-serviceplan"
-service_plan_sku_name   = "B1"
+environment                  = "Ayuda Dev"
+environment_suffix           = "-NA-CI"
+environment_suffix_lowercase = "naci"
```
Comment (serherca) [inline FSharpTemplate/Terraform/na-ci.terraform.tfvars:5]: This variable is missing in the rest of environment files.

**Comment by serherca** [comment]
Old PR that was never merged.

**Pull request closed without merging**